ui/app/components/secret-edit.js

```diff
@@ -157,6 +159,15 @@ export default Component.extend(FocusOnInsertMixin, {
     return this.secretDataIsAdvanced || this.preferAdvancedEdit;
   }),
 
+  updateWrappedSecret() {
+    this.store
+      .adapterFor('tools')
+      .toolAction('wrap', this.secretDataAsJSON, { wrapTTL: 1800 })
```

Comment:
I wonder if we want to allow them to input the ttl for wrapping too... probably a question for @joshuaogle